tools/flang1/flang1exe/gbldefs.h

```diff
@@ -143,13 +143,8 @@ typedef int LOGICAL;
 
 void finish(void); /* from main.c    */
 
-/* mall.c */
-char *sccalloc(BIGUINT64);
-void sccfree(char *);
-char *sccrelal(char *, BIGUINT64);
-#ifdef DEBUG
-void bjunk(void *p, BIGUINT64 n);
-#endif
+#include "mall.h"
+#include "listing.h"
```

Comment:
Can we avoid including these headers within `gbldefs.h`, and instead include them in the `.c` or `.cpp` files that actually need the function declarations?